test/DynamoCoreWpfTests/ViewExtensions/DocumentationBrowserViewExtensionTests.cs

```diff
@@ -142,9 +143,10 @@ public void CanCreatePackageNodeDocumentationAndLoadImages()
 
             // Assert
             Assert.IsTrue(!string.IsNullOrEmpty(browserView.VirtualFolderPath));
+            //TODO this is false because package image loading is now broken.
```

Comment:
Did you intend to remove this comment? Is it still broken? If it's, doesn't the test need to be marked as failure?

Reply:
thanks for the catch I will send another PR removing this todo.